modules/ROOT/pages/proc_assembling-a-pipeline.adoc

```diff
@@ -127,7 +127,7 @@ The execution order of the tasks is determined by the dependencies defined betwe
 . Create the `Pipeline`:
 +
 ----
-$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/pipeline/pipeline.yaml
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/04_pipeline.yaml
```

Comment:
We must point it to the release branch and not to master branch. This link should ideally be - https://raw.githubusercontent.com/openshift/pipelines-tutorial/release-v0.9/pipeline/pipeline.yaml

Note the branch - release-v0.9. The same change should be done for all the links in this PR